scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/core/db/SchedulerDBManager.java

```diff
@@ -100,7 +104,10 @@
 import com.google.common.collect.ImmutableSet;
 import com.google.common.collect.Iterables;
 
+import it.sauronsoftware.cron4j.Scheduler;
```

Comment:
We do use a pool to handle the job removal but my work was to actually remove this as we noticed is slowed the removal mechanism when the scheduler is heavily loaded (jobs were removed one by one) and replace it with a single thread doing the removal in-memory and doing some bulk operations in DB. This cron4j lib is already a dependency of the scheduler and rm server as we use it to schedule job ahead (START_AT) or in the resources deployment with the Cron policy.